docs/tutorials/excel-tutorial-create-custom-functions.md

```diff
@@ -22,7 +22,7 @@ In this tutorial, you will:
 
```

Comment:
Various pages missing ms.date update

Reply:
Not sure if I can get to all of these manually... I'll fix them if I can.